sw/airborne/modules/imu/imu.c

```diff
@@ -162,7 +162,7 @@ PRINT_CONFIG_VAR(IMU_BODY_TO_IMU_PSI)
 static void send_accel_raw(struct transport_tx *trans, struct link_device *dev)
 {
   static uint8_t id = 0;
-  pprz_msg_send_IMU_ACCEL_RAW(trans, dev, AC_ID, &imu.accels[id].abi_id,
+  pprz_msg_send_IMU_ACCEL_RAW(trans, dev, AC_ID, &imu.accels[id].abi_id, &imu.accels[id].temperature,
```

Comment:
I'm wondering if temp should be included in every message, but maybe it is fine for raw imu messages. Have you tried to make calibration including the temperature ?

Reply:
I haven't done that yet, but that was the idea indeed and was a preparation/validation for that. This way it can be used for calibration also, next to that you can see and analyse the effects of temperature on the sensors.